pinot-spi/src/main/java/org/apache/pinot/spi/utils/CommonConstants.java

```diff
@@ -492,6 +492,41 @@ public static class Broker {
         "pinot.broker.enable.dynamic.filtering.semijoin";
     public static final boolean DEFAULT_ENABLE_DYNAMIC_FILTERING_SEMI_JOIN = true;
 
+    /**
+     * Whether to use physical optimizer by default.
+     * This value can always be overridden by {@link Request.QueryOptionKey#USE_PHYSICAL_OPTIMIZER} query option
+     */
+    public static final String CONFIG_OF_USE_PHYSICAL_OPTIMIZER = "pinot.broker.use.physical.optimizer";
```

Comment:
Would recommend to keep the prefix: `pinot.broker.multistage` rather than just `pinot.broker`.

To be consistent with `CONFIG_OF_BROKER_ENABLE_MULTISTAGE_MIGRATION_METRIC`,`CONFIG_OF_SPOOLS`, etc.